.vib/owncloud/cypress/cypress/integration/owncloud_spec.js

```diff
@@ -4,77 +4,36 @@ import { random } from '../support/utils';
 it('allows creating a folder and uploading a file ', () => {
   cy.login();
   cy.get('.new').click();
-  cy.get('[data-action="folder"]').click();
+  cy.contains('Folder').click();
   cy.fixture('folders').then((folder) => {
     cy.get('[value="New folder"]').type(`${folder.newFolder.name}.${random}`);
     cy.contains('Create').click();
-    cy.contains('.innernametext', `${folder.newFolder.name}.${random}`).click();
-  });
-  cy.get('.new').click();
-  cy.get('[data-action="upload"]').click();
-  cy.get('input[type="file"]').selectFile(
-    'cypress/fixtures/file_to_upload.json',
-    { force: true }
-  );
-  cy.contains('No files in here').should('not.be.visible');
-});
-
-it('allows adding a group and a user', () => {
-  cy.login();
-  cy.visit('/index.php/settings/users');
-  cy.contains('Add Group').click();
-  cy.fixture('groups').then((group) => {
-    cy.get('#newgroupname').type(`${group.newGroup.name}.${random}`);
-    cy.get('#newgroup-form').within(() => {
-      cy.get('[type="submit"]').click();
+    cy.get('span').contains(`${folder.newFolder.name}.${random}`).click();
+    cy.contains('No files in here');
```

Comment:
I think this line does not add any value (only flakiness). I'd remove it if it is not strictly necessary.